.goreleaser-static.yml

```diff
@@ -0,0 +1,337 @@
+    main: ./cmd/apps/vpn-server/
+    ldflags: -s -w -linkmode external -extldflags '-static' -buildid= -X github.com/skycoin/skywire-utilities/pkg/buildinfo.version=v{{.Version}} -X github.com/skycoin/skywire-utilities/pkg/buildinfo.commit={{.ShortCommit}} -X github.com/skycoin/skywire-utilities/pkg/buildinfo.date={{.Date}} -X github.com/skycoin/skywire/pkg/visor.BuildTag={{.Os}}_{{.Arch}}
+
+  - id: vpn-client-amd64
+    binary: apps/vpn-client
+    goos:
+      - linux
+    goarch:
+      - amd64
+    env:
+      - CGO_ENABLED=0
+      - CC=musl-gcc
+    main: ./cmd/apps/vpn-client/
+    ldflags: -s -w -linkmode external -extldflags '-static' -buildid= -X github.com/skycoin/skywire-utilities/pkg/buildinfo.version=v{{.Version}} -X github.com/skycoin/skywire-utilities/pkg/buildinfo.commit={{.ShortCommit}} -X github.com/skycoin/skywire-utilities/pkg/buildinfo.date={{.Date}}
+
+  - id: vpn-client-arm64
+    binary: apps/vpn-client
+    goos:
+      - linux
+    goarch:
+      - arm64
+    env:
+      - CGO_ENABLED=0
+      - CC=aarch64-linux-gnu-gcc
+    main: ./cmd/apps/vpn-client/
+    ldflags: -s -w -linkmode external -extldflags '-static' -buildid= -X github.com/skycoin/skywire-utilities/pkg/buildinfo.version=v{{.Version}} -X github.com/skycoin/skywire-utilities/pkg/buildinfo.commit={{.ShortCommit}} -X github.com/skycoin/skywire-utilities/pkg/buildinfo.date={{.Date}} -X github.com/skycoin/skywire/pkg/visor.BuildTag={{.Os}}_{{.Arch}}
+
+  - id: vpn-client-arm
+    binary: apps/vpn-client
+    goos:
+      - linux
+    goarch:
+      - arm
+    goarm:
+      - 6
+    env:
+      - CGO_ENABLED=0
+      - CC=arm-linux-gnueabi-gcc
+    main: ./cmd/apps/vpn-client/
+    ldflags: -s -w -linkmode external -extldflags '-static' -buildid= -X github.com/skycoin/skywire-utilities/pkg/buildinfo.version=v{{.Version}} -X github.com/skycoin/skywire-utilities/pkg/buildinfo.commit={{.ShortCommit}} -X github.com/skycoin/skywire-utilities/pkg/buildinfo.date={{.Date}} -X github.com/skycoin/skywire/pkg/visor.BuildTag={{.Os}}_{{.Arch}}
+
+archives:
+  - id: static-amd64
+    format: tar.gz
+    wrap_in_directory: false
+    name_template: 'skywire-v{{ .Version }}-{{ .Os }}-{{ .Arch }}'
+    files:
+      - dmsghttp-config.json
+    builds:
+      - skywire-visor-amd64
+      - skywire-cli-amd64
+      - skysocks-amd64
+      - skysocks-client-amd64
+      - skychat-amd64
+      - vpn-server-amd64
+      - vpn-client-amd64
+  
+  - id: static-arm64
+    format: tar.gz
+    wrap_in_directory: false
+    name_template: 'skywire-v{{ .Version }}-{{ .Os }}-{{ .Arch }}'
+    files:
+      - dmsghttp-config.json
+    builds:
+      - skywire-visor-arm64
+      - skywire-cli-arm64
+      - skysocks-arm64
+      - skysocks-client-arm64
+      - skychat-arm64
+      - vpn-server-arm64
+      - vpn-client-arm64
+  
+  - id: static-arm
+    format: tar.gz
+    wrap_in_directory: false
+    name_template: 'skywire-v{{ .Version }}-{{ .Os }}-{{ .Arch }}'
```

Comment:
We should take out the `v`
```
   • archives         
      • creating                  archive=dist/skywire-vv0.6.0-next-linux-arm64.tar.gz
      • creating                  archive=dist/skywire-vv0.6.0-next-linux-arm.tar.gz
      • creating                  archive=dist/skywire-vv0.6.0-next-linux-amd64.tar.gz
   • creating source archive
   • linux packages   
   • snapcraft packages
   • calculating checksums
      • checksumming              file=skywire-vv0.6.0-next-linux-amd64.tar.gz
      • checksumming              file=skywire-vv0.6.0-next-linux-arm.tar.gz
      • checksumming              file=skywire-vv0.6.0-next-linux-arm64.tar.gz
```
Same for the previous ones and I'm thinking it will be same for `.goreleaser.yml`, can't test it because it's for mac.

Reply:
Its just a bug of goreleaser on showing logs. You can check released file names and their values on checksums.txt here: https://github.com/mrpalide/skywire/releases/tag/v1.0.0-rc25